packages/core/src/runtime/App.tsx

```diff
@@ -10,11 +10,13 @@ import { HelmetProvider } from 'react-helmet-async';
 import React, { useContext, useLayoutEffect } from 'react';
 import { Header, PageData, cleanUrl, MDX_REGEXP } from '@rspress/shared';
 import globalComponents from 'virtual-global-components';
-import 'virtual-global-styles';
 
 // eslint-disable-next-line import/no-commonjs
 const { default: Theme } = require('@theme');
 
+// Inject the global styles come from plugins or config file
+require('virtual-global-styles');
```

Comment:
I'm curious why should we use `require` here..

Reply:
it's not necessary to use require and i will transform it into import statement